rules/sort-object-types.ts

```diff
@@ -273,8 +275,9 @@ export let sortObjectTypeElements = <MessageIds extends string>({
       }
 
       if (
-        !selectors.includes('index-signature') &&
-        !selectors.includes('method')
+        !(['index-signature', 'method'] as Selector[]).some(selector =>
```

Comment:
[readability] 

1. If we keep this:

`const` is better: no type casting!

```suggestion
        !(['index-signature', 'method'] as const).some(selector =>
```

2. However, my brain finds it quite harder to understand

```ts
if (
  !(['index-signature', 'method'] as const).some(selector =>
    selectors.includes(selector),
  )
)
```

than two simple negations.

```ts
if (
  !selectors.includes('index-signature') &&
  !selectors.includes('method')
) {
```

Reply:
Improved.